tests/test_flow.py

```diff
@@ -460,8 +460,8 @@ def test_map_with_state_flow():
 
 def test_map_with_cache_state_flow():
     table_object = Table("table", NoopDriver())
-    table_object.set_static_attrs('tal', {'color': 'blue'})
```

Comment:
In the tests you can use `table_object['tal'] = {'color': 'blue'}` as it was before 😄